### PR TITLE
Manually compile template, disabling as-sortable-* directives before rendering to DOM

### PR DESCRIPTION
**Pull request by benjaminapetersen**
@rhamilto check this out and let me know what you think.  Essentially its just a find and replace of the `as-sortable` string in the template, but done before it is rendered to the DOM.  I'm replacing it with `as-sortable-DISABLED` instead of removing so that the `as-sortable="dragControlListeners"` doesn't end up as `="dragControlListeners"` which would be invalid html. There is probably a way to eliminate it entirely, but I don't know if it matters much.

At the end of the `link()` function, this line `elem.append($compile(tpl)($scope));` manually compiles the template and renders it into the DOM.

### CODE REVIEW

File: src/directives/key-value-editor.js [hunk 1]
```diff
@@ -110,11 +122,11 @@
 
               };
             }
-          ],
+          ]
           // as a fn in case we want to allow configurable templates
```
Comment (rhamilto) [inline src/directives/key-value-editor.js]: Any reason you didn't delete these lines?

**Comment by benjaminapetersen** [comment]
Down side to this approach is that it won't update... if you had a way to flip `cannot-sort="true||false"` to toggle, this would miss it. Still, is a step in the right direction, and if we think toggling that attr on the fly is unnecessary may not need to do anything else.

**Comment by rhamilto** [comment]
Neat!

Aare there issues leaving as-sortable-DISABLED, as-sortable-item, and as-sortable-item-handle in the markup?

**Comment by rhamilto** [comment]
> if we think toggling that attr on the fly is unnecessary

I don't know of a use case where we'd need it, so I think it's fine for now.

File: src/directives/key-value-editor.js (hunk 1 above)
Comment (benjaminapetersen) [inline src/directives/key-value-editor.js]: Mostly contemplating... prob should delete, I'll do that.

**Comment by benjaminapetersen** [comment]
I dont believe so re the `*-DISABLED`attributes.  They should just end up not matching any registered directive and doing nothing. It might be a better practice to remove them.  Still thinking about it, but I may merge this so we can keep working with a simpler template and follow with an improvement on that later.

**Pull request merged**